base_wamas_ubl/lib/wamas/ubl_template/picking.xml

```diff
@@ -0,0 +1,95 @@
+				<cbc:Telephone><t t-raw="extra_data['DespatchSupplierParty']['Contact.Telephone']"/></cbc:Telephone>
+				<cbc:ElectronicMail><t t-raw="extra_data['DespatchSupplierParty']['Contact.ElectronicMail']"/></cbc:ElectronicMail>
+			</cac:Contact>
+		</cac:Party>
+	</cac:DespatchSupplierParty>
+	<cac:DeliveryCustomerParty>
+		<cac:Party>
+            <cac:PartyIdentification>
+                <cbc:ID><t t-raw="record.IvAusk_KST_KuNr"/></cbc:ID>
+            </cac:PartyIdentification>
+			<cac:PartyName>
+				<cbc:Name><t t-raw="extra_data['DeliveryCustomerParty']['PartyName']"/></cbc:Name>
+			</cac:PartyName>
+			<cac:PostalAddress>
+				<cbc:StreetName><t t-raw="extra_data['DeliveryCustomerParty']['StreetName']"/></cbc:StreetName>
+				<cbc:CityName><t t-raw="extra_data['DeliveryCustomerParty']['CityName']"/></cbc:CityName>
+				<cbc:PostalZone><t t-raw="extra_data['DeliveryCustomerParty']['PostalZone']"/></cbc:PostalZone>
+				<cbc:CountrySubentity><t t-raw="extra_data['DeliveryCustomerParty']['CountrySubentity']"/></cbc:CountrySubentity>
+				<cac:Country>
+					<cbc:IdentificationCode><t t-raw="extra_data['DeliveryCustomerParty']['IdentificationCode']"/></cbc:IdentificationCode>
+				</cac:Country>
+			</cac:PostalAddress>
+			<cac:PartyTaxScheme>
+				<cbc:CompanyID><t t-raw="extra_data['DeliveryCustomerParty']['CompanyID']"/></cbc:CompanyID>
+				<cac:TaxScheme>
+					<cbc:ID><t t-raw="extra_data['DeliveryCustomerParty']['TaxScheme.ID']"/></cbc:ID>
+					<cbc:TaxTypeCode><t t-raw="extra_data['DeliveryCustomerParty']['TaxScheme.TaxTypeCode']"/></cbc:TaxTypeCode>
+				</cac:TaxScheme>
+			</cac:PartyTaxScheme>
+			<cac:Contact>
+				<cbc:Name><t t-raw="extra_data['DeliveryCustomerParty']['Contact.Name']"/></cbc:Name>
+				<cbc:Telephone><t t-raw="extra_data['DeliveryCustomerParty']['Contact.Telephone']"/></cbc:Telephone>
+				<cbc:Telefax><t t-raw="extra_data['DeliveryCustomerParty']['Contact.Telefax']"/></cbc:Telefax>
+				<cbc:ElectronicMail><t t-raw="extra_data['DeliveryCustomerParty']['Contact.ElectronicMail']"/></cbc:ElectronicMail>
+			</cac:Contact>
+		</cac:Party>
+	</cac:DeliveryCustomerParty>
+	<cac:DespatchLine t-foreach="record.lines" t-as="line">
+		<cac:Shipment>
+			<cac:TransportHandlingUnit>
+				<cac:ActualPackage>
+					<cbc:ID><t t-raw="line.package.IvTek_TeId"/></cbc:ID>
+				</cac:ActualPackage>
+				<cbc:TransportHandlingUnitTypeCode><t t-raw="line.package.IvTts_THM_ThmId"/></cbc:TransportHandlingUnitTypeCode>
+			</cac:TransportHandlingUnit>
+			<cac:GrossWeightMeasure>
+				<cbc:Measure><t t-raw="line.package.IvTek_GesGew"/></cbc:Measure>
+			</cac:GrossWeightMeasure>
+		</cac:Shipment>
+		<cbc:ID></cbc:ID>
+        <cbc:DeliveredQuantity t-att-unitCode="MAPPING['unitCode'][line.IvArt_HOSTUNITS_HostEinh] or MAPPING['unitCode'][line.IvArt_Bestand_Einheit]"><t t-raw="line.Mngs_Mng"/></cbc:DeliveredQuantity>
+		<cac:OrderLineReference>
+			<cbc:LineID></cbc:LineID>
```

Comment:
we can probably use `<t t-raw="record_index"/>` here as well to get a valid value